bibtexparser/bparser.py

```diff
@@ -65,7 +65,8 @@ def __init__(self, data=None,
                  customization=None,
                  ignore_nonstandard_types=True,
                  homogenize_fields=False,
-                 common_strings=False):
+                 common_strings=False,
+                 add_raw_bibtex_entry=False):
```

Comment:
In that file. Look at the top of the line `self.common_strings = common_strings` to see what I mean.

If you would like to add a paragraph to the tutorial, it's definitely welcome :)